wait-for.bats

```diff
@@ -79,7 +79,7 @@
 	sleep $((timeout + 1))
 	EOF
   chmod +x delay
-  nc -lk -p 80 -e $(pwd)/delay & ncpid=$!
+  ncat -lk -p 80 -e $(pwd)/delay & ncpid=$!
```

Comment:
This probably breaks the tests when running them without the Docker container. I'm considering some kind of lookup or "fallback" mechanism. Maybe the test-env container should override the default `nc` binary with the `nmap-ncat`. What do you think?

Reply:
> However, do we really need it? Inside Alpine it also works if we only install nmap-ncat. Inside the Node.js image I was already working, thus no need to install different dependencies, right?

You're right and I also can't reproduce the issue. This is probably an example for the classic "don't change too much at the same time"... 😬 😉

Reply:
I have cherry-picked your commit, so this is back to the previous `nc` command.